libs/designer-ui/src/lib/monitoring/values/keyvaluepairs.tsx

```diff
@@ -57,14 +57,14 @@ export const KeyValuePairs: React.FC<ValueProps> = ({ displayName, value = {}, v
       <label className="msla-trace-value-display-name">{displayName}</label>
       <div className="msla-trace-value-text msla-trace-value-key-value-pairs">
         <DetailsList
-          ariaLabel={displayName}
+          ariaLabelForGrid={displayName}
           ariaLabelForListHeader={displayName}
           checkboxVisibility={CheckboxVisibility.hidden}
           columns={columns}
           compact={true}
           isHeaderVisible={true}
           items={items}
-          layoutMode={DetailsListLayoutMode.justified}
+          layoutMode={DetailsListLayoutMode.fixedColumns}
```

Comment:
Hmmmm this is needed as well? Can you add screenshots here?

Reply:
We're unable to resize the detailsList columns in standalone (not too sure why), but with this property we don't have well-spaced column widths on load 
before:
![before](https://github.com/user-attachments/assets/3dfbb27d-461e-40a2-ae89-b80c5d5a0d5e)

after:
![after](https://github.com/user-attachments/assets/c4c0faea-9d56-4921-82c2-65b98172f83e)